common/capture.go

```diff
@@ -34,9 +34,9 @@ var (
 )
 
 func initCaptureTypes() {
-	// add ovs type
 	CaptureTypes["ovsbridge"] = CaptureType{Allowed: []string{"ovssflow", "pcapsocket"}, Default: "ovssflow"}
 	CaptureTypes["device"] = CaptureType{Allowed: []string{"afpacket", "pcap", "pcapsocket", "sflow"}, Default: "afpacket"}
+	CaptureTypes["dpdk"] = CaptureType{Allowed: []string{"dpdk"}, Default: "dpdk"}
```

Comment:
you need to modify the webui too otherwise it is impossible to start a capture from it